### ISSUE TITLE
Riddled with errors!

### ISSUE THREAD

**Issue by lewisj4** [lewisj4/suite_solo_app#3]
@williamfshaw @bigtone1284 @h5w4
I'm trying to nodemon server.js but I am getting a weird TypeError that is saying "TypeError: Router.use() requires middleware function but got a Object"
The rest of the error trail is
    at Function.<anonymous> (/Users/jameslewis/WDI_main/suite_solo_app/node_modules/express/lib/router/index.js:446:13)
    at Array.forEach (native)
    at Function.use (/Users/jameslewis/WDI_main/suite_solo_app/node_modules/express/lib/router/index.js:444:13)
    at EventEmitter.<anonymous> (/Users/jameslewis/WDI_main/suite_solo_app/node_modules/express/lib/application.js:216:21)
    at Array.forEach (native)
    at EventEmitter.use (/Users/jameslewis/WDI_main/suite_solo_app/node_modules/express/lib/application.js:213:7)
    at Object.<anonymous> (/Users/jameslewis/WDI_main/suite_solo_app/server.js:33:5)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)

I have narrowed it down to what I believe is an express issue, but I am not exactly sure why I am getting it. My server.js file looks like
```var express              =require('express');
var bodyParser           =require('body-parser');
var path                 =require('path');
var logger               =require('morgan');
var session              =require('express-session');
var bcrypt               =require('bcrypt');
var allergyRoutes        =require('./routers/allergy_routes.js');
var immunizationRoutes   =require('./routers/immunization_routes.js');
var diagnosisRoutes      =require('./routers/diagnosis_routes.js');
var medicationRoutes     =require('./routers/medication_routes.js');
var surgeryRoutes        =require('./routers/surgery_routes.js');
var userRoutes           =require('./routers/user_routes.js');

var app = express(); 

app.use( logger('dev') );
app.use( bodyParser.urlencoded({ extended: false }) );
app.use( bodyParser.json() );
app.use( session({
    secret: 'ballersballin',
    resave: false,
    saveUninitialized: true
}));

app.use(express.static('./public'));

app.use('/allergies', allergyRoutes);
app.use('/immunizations', immunizationRoutes);
app.use('/diagnoses', diagnosisRoutes);
app.use('/medications', medicationRoutes);
app.use('/surgeries', surgeryRoutes);
app.use('/users', userRoutes);

app.listen(3000, function() {
    console.log('Server listening on 3000...');
});

module.exports = app;```

**Comment by bigtone1284** [comment]
@h4w5

**Comment by bigtone1284** [comment]
@lewisj4

Please leave an issue from within your projet's repo.  I'd like to look over some more code from your app. Also, please push your code to your repo.

**Comment by lewisj4** [comment]
@bigtone1284 
So I fixed the error that I was getting, silly me forgot a line of code. Anyway, I'm still kind of confused on the trail you were leading me down yesterday afternoon for a separate seed file that builds the relationships. I understand the function you wrote down, but I feel like that function hinders on relationships already existing; and because I separated my users, surgeries, etc into separate variables in my first seed file those relationships haven't been established yet. I tried doing some things last night but none of them worked out.
